### ISSUE TITLE
[ICC] Sindragosa Fight bug causing the fight and ICC raid to be impossible, Kirin Tor Realm

### ISSUE THREAD

**Issue by Boberthy** [dalaranwow/dalaran-wow#5679]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

The Sindragosa fight in Icecrown Citadel is impossible to complete, due to the second and third phases, [Frost Beacon] will be cat and entomb players in [Ice Tomb] this is normal, however the ice tomb's cannot be damaged as they are classified as being "not in line of sight" for some reason, meaning they only thing that can happen is the players inside the tomb have to be left to suffocate and die making the fight impossible to finish and in turn makes the entire raid impossible to finish.
I'd like to also add this is on the Kirin Tor realm.

**Expected behaviour**: Tell us what should happen instead

What should happen is during the encounter ice tombs should be attackable and not considered "out of line of sight" at any point during the fight allowing trapped players to be freed instead of being encased in there till they suffocate or the encounter ends.

**Steps to reproduce the problem**:

1. Start the encounter with Sindragosa in Icecrown Citadel 
2. fight until the second phase
3. Frost beacon will be cast on two random players and entomb them 

**Include proofs for this behaviour**

http://wowwiki.wikia.com/wiki/Sindragosa_(tactics) 
-"Players are freed from the Ice Tombs by destroying them (~450k HP)."

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

npc=36853 =Sindragosa
spell=70126 = Frost Beacon
spell=70157 = Ice Tomb
Zone ID: 4812 = Icecrown Citadel

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by MalkaviaDW** [comment]
#5161
